app/services/hyrax/embargo_manager.rb

```diff
@@ -17,6 +17,7 @@ module Hyrax
   #    - "Enforced" means the object's visibility matches the pre-release
   #      visibility of the embargo; i.e. the embargo has been applied,
   #      but not released.
+  #      "Deactivate" means that the existing embargo will be removed
```

Comment:
Minor syntax consistency adjustment 

```suggestion
  #    - "Deactivate" means that the existing embargo will be removed
```